util/components.js

```diff
@@ -17,3 +19,86 @@ export async function setOEBInputValueByCSS(driver, cssSelector, value) {
   await oebInputChild.clear()
   await oebInputChild.sendKeys(value);
 }
+
+export async function addNewTag(driver, tagName ){
+  await driver.wait(until.elementLocated(By.css('input[placeholder="Neuer Tag..."]')), defaultWait);
```

Comment:
Please consistently indent with 4 spaces instead of 2.

Reply:
I updated the indent, but it will be back to 2 if we decide to use `prettier`